llm_client/llm_api_client/anthropic_client.py

```diff
@@ -21,13 +20,15 @@ def __init__(self, config: LLMAPIClientConfig):
         self._headers[ACCEPT_HEADER] = ACCEPT_VALUE
         self._headers[AUTH_HEADER] = self._api_key
 
-    async def text_completion(self, prompt: str, model: str | None = None, max_tokens: int | None = None, **kwargs) ->\
+    async def text_completion(self, prompt: str, model: str | None = None, max_tokens: int | None = None, temperature: float = 0,
+                              **kwargs) -> \
             list[str]:
         if max_tokens is None:
             raise ValueError("max_tokens must be specified")
```

Comment:
add here if max_tokens is None or kwargs["max_tokens_to_sample"]: raise ValueError("max_tokens must be specified")

Reply:
i think adding kwargs["max_tokens_to_sample"] is not the best , i would rather do : if max_tokens is None and kwargs["max_tokens_to_sample"] is None

Reply:
that is what I meant